packages/react-storage/src/components/StorageBrowser/views/LocationActionView/constants.ts

```diff
@@ -14,6 +15,7 @@ export const STATUS_DISPLAY_VALUES = {
   CANCELED: 'Canceled',
   COMPLETE: 'Completed',
   FAILED: 'Failed',
+  OVERWRITE_PREVENTED: 'Overwrite Prevented',
```

Comment:
Overwrite prevention is specific to uploads and should not be included in the generic statuses

Reply:
But the status is generic to the status column, no?

Reply:
If we add it to the status column then we need to add it to the status counts and allow for conditionally hiding that column as it is not pertinent to most actions. IMO we should just be returning an overwrite prevention specific message from the `upload` handler on failure